git_tracker_name_translator.go

```diff
@@ -0,0 +1,14 @@
+package storybranch
+
+import (
+	"regexp"
+	"strconv"
+)
+
+// GetPivotalTrackerTaskID returns a string containing the task ID of the current branch name of the repository.
+func GetPivotalTrackerTaskID(branchName string) int {
+	re := regexp.MustCompile(`[1-9]+$`)
```

Comment:
It just occurred to me that a zero probably wouldn't be accepted here. Perhaps `\d+$`?